src/Gui/NaviCube.cpp

```diff
@@ -380,45 +409,41 @@ GLuint NaviCubeImplementation::createCubeFaceTex(QtGLWidget* gl, float gap, cons
 		QPainterPath pathSquare;
 		pathSquare.addRect(QRectF(gapi, gapi, (qreal)texSize - 2.0 * gapi, (qreal)texSize - 2.0 * gapi));
 		paint.fillPath(pathSquare, Qt::white);
-		paint.setPen(pen);
-		paint.drawPath(pathSquare);
 	}
 	else if (shape == SHAPE_CORNER) {
 		QPainterPath pathCorner;
-		QRectF rectCorner = QRectF(3.46 * gapi, 3.31 * gapi, sqrt(2) * gapi, 1.3 * gapi);
-		pathCorner.moveTo(rectCorner.left() + (rectCorner.width() / 2), rectCorner.top());
-		pathCorner.lineTo(rectCorner.bottomLeft());
-		pathCorner.lineTo(rectCorner.bottomRight());
-		pathCorner.lineTo(rectCorner.left() + (rectCorner.width() / 2), rectCorner.top());
+		QRectF rectCorner = QRectF(3.5 * gapi, 3.5 * gapi, 1.5 * gapi, 1.5 * gapi);
```

Comment:
This does not seem to work if gap argument is changed. Maybe change to:
`QRectF rectCorner = QRectF((qreal)texSize / 2.0 - 0.75 * gapi, (qreal)texSize / 2.0 - 0.75 * gapi, 1.5 * gapi, 1.5 * gapi);`

Reply:
This does not work for me, I get then this result:
![FreeCAD_Y3VCpFQjE2](https://user-images.githubusercontent.com/1828501/167990652-3c76e7a1-90d7-4acb-847c-5cba903bc8d7.png)

Reply:
The change presumed the round 1.5 gap diameter corners. With triangular it should be:

Based on the literal 3.41 that feels upside down:
`QRectF(((qreal)texSize  - sqrt(2) * gapi) / 2, ((qreal)texSize  - sqrt(8/3) * gapi) / 2, sqrt(2) * gapi, sqrt(1.5) * gapi)`

What I would have calculated if I didn't know that 3.41 seems to be almost right:
`QRectF(((qreal)texSize  - sqrt(2) * gapi) / 2, ((qreal)texSize  - sqrt(2/3) * gapi) / 2, sqrt(2) * gapi, sqrt(1.5) * gapi)`

Hope this works. I'm not able to test by myself. I don't understand why the lines are thicker.

Reply:
The first one leads to:
![FreeCAD_Fn264SjFqk](https://user-images.githubusercontent.com/1828501/168060117-f1459ade-ba32-45fb-8133-a87d5a1c4f10.png)

the second one to:
![FreeCAD_rw9TBWqcwE](https://user-images.githubusercontent.com/1828501/168058856-172be054-708f-4610-b7ab-92f123bec373.png)